syntax = "proto3";

package example;
option cc_generic_services=true;

service EchoServiceRpc {
  rpc Echo (EchoRequest) returns (EchoResponse);
}

message EchoRequest {
  string msg = 1;
}

message EchoResponse {
  string msg = 1;
}
